src/NServiceBus.AcceptanceTests/Sagas/When_replies_to_message_published_by_a_saga.cs

```diff
@@ -93,8 +93,7 @@ public class ReplyToPubMsgSaga : Saga<ReplyToPubMsgSaga.ReplyToPubMsgSagaData>,
                 public Task Handle(StartSaga message)
                 {
                     Data.DataId = message.DataId;
-                    Bus.Publish(new DidSomething { DataId = message.DataId });
-                    return Task.FromResult(0);
+                    return Bus.PublishAsync(new DidSomething { DataId = message.DataId });
```

Comment:
Really looks funky to return the publish like this (I know it works and is terse), I can see a user asking "how can I do multiple bus.Publish?"

Is 

```
await Bus.PublishAsync(new DidSomething { DataId = message.DataId });
return Task.FromResult(0);
```

an option?